cam2imageGUI.py

```diff
@@ -39,7 +39,9 @@
 }
 
 
+#####################
 ### Error Windows ###
+#####################
 
 def missingPath():
     layout = [[sg.Text("Path not propperly chosen")], [sg.Button("OK")]]

```

Comment:
what is layout doing? this function is not doing "anything" if you exclude side effect (which should not be used)